pointcloud.js

```diff
@@ -435,9 +480,6 @@ proto.update = function(options) {
 
   //Create new buffers
   var points = options.position
-  if(!points) {
-    return
```

Comment:
I think we shouldn't skip in this way here that could initiate various webgl and indexing issues.

Reply:
Cool. Could you provide an example of where this break?